extensions/ql-vscode/src/view/data-extensions-editor/MethodRow.tsx

```diff
@@ -197,6 +221,12 @@ function ModelableMethodRow(props: Props) {
           </UsagesButton>
         )}
         <ViewLink onClick={jumpToUsage}>View</ViewLink>
+        {!inSource && (
+          <ClassificationsContainer>
+            {inTest && <VSCodeTag>Test</VSCodeTag>}
```

Comment:
Random thought. Is it possible to put a tooltip / title text on these that explains "This method is only used from test code"?